ml-agents/mlagents/trainers/learn.py

```diff
@@ -199,26 +194,20 @@ def write_timing_tree(output_dir: str) -> None:
         )
 
 
-def create_sampler_manager(sampler_config, run_seed=None):
-    resample_interval = None
+def maybe_add_samplers(
+    sampler_config: Optional[Dict], env: SubprocessEnvManager
+) -> None:
+    restructured_sampler_config: Dict[str, List[float]] = {}
+    # TODO send seed
     if sampler_config is not None:
         if "resampling-interval" in sampler_config:
-            # Filter arguments that do not exist in the environment
-            resample_interval = sampler_config.pop("resampling-interval")
-            if (resample_interval <= 0) or (not isinstance(resample_interval, int)):
-                raise SamplerException(
-                    "Specified resampling-interval is not valid. Please provide"
-                    " a positive integer value for resampling-interval"
-                )
-
-        else:
-            raise SamplerException(
-                "Resampling interval was not specified in the sampler file."
-                " Please specify it with the 'resampling-interval' key in the sampler config file."
+            logger.warning(
+                "The resampling-interval is no longer necessary for parameter randomization. It is being ignored."
             )
-
-    sampler_manager = SamplerManager(sampler_config, run_seed)
-    return sampler_manager, resample_interval
+            sampler_config.pop("resampling-interval")
+        for param, config in sampler_config.items():
+            restructured_sampler_config[param] = config
+        env.reset(config=restructured_sampler_config)
```

Comment:
I'm just using the control already in place in the env_manager. I can trigger this message passing/reset from elsewhere but that would require passing sampler info around.